src/components/LooMap/ToiletMarkerIcon.tsx

```diff
@@ -1,13 +1,20 @@
 import L from 'leaflet';
 
-const ICON_DIMENSIONS = [22, 34];
-const LARGE_ICON_DIMENSSIONS = ICON_DIMENSIONS.map((i) => i * 1.5);
-const getIconAnchor = (dimensions: number[]) => [
+const ICON_DIMENSIONS: [number, number] = [22, 34];
+const LARGE_ICON_DIMENSIONS: [number, number] = ICON_DIMENSIONS.map(
```

Comment:
To get the narrowest types!

```suggestion
const ICON_DIMENSIONS = [22, 34] as const;
const LARGE_ICON_DIMENSIONS = [33, 51] as const
```